moveit_ros/moveit_servo/test/test_servo_interface.cpp

```diff
@@ -162,7 +165,7 @@ TEST_F(ServoFixture, SendJointServoTest)
   log_time_start = node_->now();
 
   // Allow the last command to go stale and measure the output position
-  const int sleep_time = 1.5 * 1000 * parameters_->incoming_command_timeout;
+  const int sleep_time = 1.5 * 1000 * servo_parameters_->incoming_command_timeout;
```

Comment:
nit: using seconds would give you a `duration` in exchange for a `1000 *` and a `milli`. Or maybe even nicer to read would be `rclcpp::sleep_for(1.5s *  servo_parameters_->incoming_command_timeout);`...

Reply:
I agree, I'll clean this up in a follow-on PR.  I am going to remove the rename.